src/components/onboarding/Intro/index.tsx

```diff
@@ -1,27 +1,155 @@
-import TempImage from '@/public/assets/graphics/cat404.png';
-import Image from 'next/image';
+import Allocation from '@/public/assets/graphics/onboarding/ACM_Fall23Allocation-JustinLu.jpg';
+import HoldBoba from '@/public/assets/graphics/onboarding/ACM_Fall24BitByteInfo_1-JustinLu.jpg';
+import Murou from '@/public/assets/graphics/onboarding/ACM_Fall24BitByteInfo_2-JustinLu.jpg';
+import Bonfire from '@/public/assets/graphics/onboarding/ACM_Fall24Bonfire_1-JustinLu.jpg';
+import RaymondBack from '@/public/assets/graphics/onboarding/ACM_Fall24Bonfire_2-JustinLu.jpg';
+import TiltKickoff from '@/public/assets/graphics/onboarding/ACM_Fall24Kickoff_1-JustinLu.jpg';
+import FocusPerson from '@/public/assets/graphics/onboarding/ACM_Fall24Kickoff_2-JustinLu.jpg';
+import KickoffBig from '@/public/assets/graphics/onboarding/ACM_Fall24Kickoff_3-JustinLu.jpg';
+import Image, { StaticImageData } from 'next/image';
+import { useEffect, useRef, useState } from 'react';
 import styles from './style.module.scss';
 
+type IntroImage = {
+  src: StaticImageData;
+  alt: string;
+  desktopSize: [x: number, y: number, width: number, height: number];
+  mobileSize: [x: number, y: number, width: number, height: number];
+  round?: boolean;
+};
+
+// https://www.figma.com/design/GiWZdbzJ2uxyknpCrB9UK6/acm-onboarding
+const images: IntroImage[] = [
+  {
+    src: FocusPerson,
+    alt: 'A student smiling in a crowd',
+    desktopSize: [89, 46, 168, 168],
+    mobileSize: [24, 326, 109, 109],
+    round: true,
+  },
```

Comment:
nit: Let's make these typed so it's easier to read what number corresponds to what

Reply:
I'm assuming you mean to change this into an object with named properties rather than a tuple